.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,54 @@
+name: .NET Core Test and Publish
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+jobs:
+
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.0.100
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
+    - name: Check format
+      run: |
+        dotnet tool install --tool-path ./ dotnet-format
+        ./dotnet-format --check --dry-run -v diagnostic
+    - name: Test
+      run: |
+        find tests -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
+        dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=lcov /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage/lcov
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  Publish:
+    if: github.ref == 'refs/heads/master' && startsWith(github.repository, 'neo-project/')
+    needs: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.0.100
+    - name: Setup NuGet.exe for use with actions
+      uses: NuGet/setup-nuget@v1
+    - name: Pack with dotnet
+      run: dotnet pack -c Debug -o out --include-source --version-suffix CI$(date +%s)
+    - name: Publish to Github Packages
+      run: |
+        nuget source Add -Name "GitHub" -Source "https://nuget.pkg.github.com/neo-project/index.json" -UserName neo-project -Password ${GITHUB_TOKEN}
+        nuget push out/*.nupkg -Source "GitHub"
```

Comment:
Did you figure out how to provide anonymous access to github package registry?

Reply:
No. I will disable it before merging.

Reply:
It is very strange that it was successfully published yesterday but it has failed today.

```
Package source with Name: GitHub added successfully.
Pushing Neo.VM.3.0.0-CI00207.nupkg to 'https://nuget.pkg.github.com/neo-project'...
  PUT https://nuget.pkg.github.com/neo-project/
An error was encountered when fetching 'PUT https://nuget.pkg.github.com/neo-project/'. The request will now be retried.
An error occurred while sending the request.
  The server returned an invalid or unrecognized response.
  PUT https://nuget.pkg.github.com/neo-project/
An error was encountered when fetching 'PUT https://nuget.pkg.github.com/neo-project/'. The request will now be retried.
An error occurred while sending the request.
  The server returned an invalid or unrecognized response.
  PUT https://nuget.pkg.github.com/neo-project/
An error occurred while sending the request.
  The server returned an invalid or unrecognized response.
##[error]Process completed with exit code 1.
```

Reply:
I tested it again and got these responses:

```
Package source with Name: GitHub added successfully.
Pushing Neo.VM.3.0.0-CI00208.nupkg to 'https://nuget.pkg.github.com/neo-project'...
  PUT https://nuget.pkg.github.com/neo-project/
An error was encountered when fetching 'PUT https://nuget.pkg.github.com/neo-project/'. The request will now be retried.
An error occurred while sending the request.
  The server returned an invalid or unrecognized response.
  PUT https://nuget.pkg.github.com/neo-project/
An error was encountered when fetching 'PUT https://nuget.pkg.github.com/neo-project/'. The request will now be retried.
An error occurred while sending the request.
  The server returned an invalid or unrecognized response.
  PUT https://nuget.pkg.github.com/neo-project/
  OK https://nuget.pkg.github.com/neo-project/ 1344ms
Your package was pushed.
```

Seems that the service of Github Packages is not stable.

Reply:
I modified the workflow. It will now automatically publish to Github and MyGet at the same time.